src/lib.rs

```diff
@@ -213,6 +213,7 @@ macro_rules! v1_x_reader_writer {
             lazy::r#struct::{LazyStruct, LazyField},
             lazy::sequence::{LazyList, LazySExp},
             lazy::encoder::value_writer::{ValueWriter, StructWriter, SequenceWriter, EExpWriter},
+            lazy::any_encoding::IonEncoding,
```

Comment:
🗺️ This re-export is only visible when the `experimental-reader-writer` feature is enabled.